### PR TITLE
feature/frontend_exam into develop

### PR DESCRIPTION
**Pull request by dhelmert**
### Reason for this PR
- Exams are now part of UC4, and need an api

### Changes in this PR
- Added exam-management.yaml
- Added examresult-management.yaml
- Updated admission-management to include exam admissions

### CODE REVIEW

**Review by RBeierling — APPROVED** [review]

File: exam-management.yaml [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+              schema:
+                $ref: '#/components/schemas/UnsignedProposal'
+              examples:
+                UnsignedProposalAddEntry:
+                  $ref: '#/components/examples/ExampleUnsignedProposal'
+          headers:
+            Location:
+              schema:
+                type: string
+                description: "URI of the newly created Admission"
```
Comment (bastihav) [inline exam-management.yaml]:
```suggestion
```

File: examresult-management.yaml [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+              schema:
+                $ref: '#/components/schemas/UnsignedProposal'
+              examples:
+                UnsignedProposalAddEntry:
+                  $ref: '#/components/examples/ExampleUnsignedProposal'
+          headers:
+            Location:
+              schema:
+                type: string
+                description: "URI of the newly created Admission"
```
Comment (bastihav) [inline examresult-management.yaml]:
```suggestion
```

**Review by bastihav — CHANGES_REQUESTED** [review]
- [x] for all lists parameters: specify that we need a comma seperated value
- [x] example description for UnsignedProposals are wrong (admission, exam, examresult)
- [x] specify "admittableAt" and "droppableAt" format -> ISO
- [x] specify admittableUntil and droppableUntil format in schema -> date iso

**Review by bastihav — APPROVED** [review]

**Review by NikoBergemann — COMMENTED** [review]
Nothing out of the ordinary

**Review by bastihav — CHANGES_REQUESTED** [review]
we now have two superflous endpoints (for resolving single enrollmentId and single username respectively), which should be removed

**Comment by dhelmert** [comment]
> 
> 
> we now have two superflous endpoints (for resolving single enrollmentId and single username respectively), which should be removed

Good point, the endpoints in certificate are now removed

File: exam-management.yaml [hunk 3]
```diff
@@ -0,0 +1,310 @@
+openapi: "3.0.0"
+info:
+  description: "This is the Exam API for UC4."
+  version: "0.16.1"
```
Comment (bastihav) [inline exam-management.yaml]:
```suggestion
  version: "0.17.1"
```

File: exam-result-management.yaml [hunk 4]
```diff
@@ -0,0 +1,266 @@
+openapi: "3.0.0"
+info:
+  description: "This is the Exam Result API for UC4."
+  version: "0.16.1"
```
Comment (bastihav) [inline exam-result-management.yaml]:
```suggestion
  version: "0.17.1"
```

**Review by bastihav — APPROVED** [review]

File: admission-management.yaml [hunk 5]
```diff
@@ -1,7 +1,7 @@
 openapi: "3.0.0"
 info:
   description: "This is the Admission API for UC4."
-  version: "0.15.1"
+  version: "0.16.1"
```
Comment (bastihav) [inline admission-management.yaml]:
```suggestion
  version: "0.17.1"
```

**Review by bastihav — APPROVED** [review]

**Pull request merged**